src/components/dialog/options.js

```diff
@@ -0,0 +1,17 @@
+import { translate } from 'react-i18next';
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+import { Button, RedButton } from '../toolbox/buttons/button';
+
+import styles from './options.css';
+
+const Options = ({ firstButton, secondButton, text }) =>
```

Comment:
I think it will be clearer if these options default to expected object to be passed, to expose what the component needs to receive.
```
{ firstButton: { text: '', onClickHandler: () => {}}, ...}
```
It could be using children props, to reduce the amount of arguments passed and make the component more flexible.

Reply:
Ther is no children component involved there. These buttons are passed as props in action